src/components/Question.js

```diff
@@ -3,64 +3,140 @@ import {
   IconButton,
   Typography,
   Button,
-  Box
+  InputLabel,
+  Box,
+  Select,
+  MenuItem,
+  FormControl,
+  TextField
 } from "@mui/material";
 import DeleteIcon from '@mui/icons-material/Delete';
 import EditIcon from '@mui/icons-material/Edit';
 
-const Question = ({ question, onDelete, onEdit }) => {
+const Question = ({ question, display, onDelete, onEdit, onAnswer }) => {
+  if (display) {
```

Comment:
Not how you do this, move the "if" into the markup below

Reply:
What is wrong with this method? It works as expected?

Reply:
It's poor code style but you don't have to change it if you don't want to...